pipeline-maven/src/test/java/org/jenkinsci/plugins/pipeline/maven/DependencyGraphTest.java

```diff
@@ -504,6 +507,7 @@ public void verify_nbm_downstream_simple_pipeline_trigger() throws Exception {
     @Test
     @EnabledOnOs(OS.LINUX) // Docker does not work on Windows 2019 servers CI agents
     public void verify_docker_downstream_simple_pipeline_trigger() throws Exception {
+        assumeTrue(Files.exists(Paths.get("/var/run/docker.sock")));
```

Comment:
https://github.com/jenkinsci/hashicorp-vault-plugin/blob/master/src/test/java/com/datapipe/jenkins/vault/util/VaultTestUtil.java#L42-L48

Reply:
Thanks for the link, @jetersen! I do not see an action item for me in this PR.